sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala

```diff
@@ -438,7 +440,7 @@ case class DataSource(
    * Writes the given [[LogicalPlan]] out in this [[FileFormat]].
```

Comment:
nit: the comment seems not accurate. It doesn't write the logical plan out in fact.